client/src/components/Constants.jsx

```diff
@@ -1,28 +1,94 @@
 import styled from 'styled-components';
 
 const margins = {
-  width: `5vw`,
   barWidth: `150`,
-  barHeight: `5px`
+  modalBarWidth: `75`,
+  barHeight: `4px`,
+  imageSize: `20px`,
+  modalImageSize: `30px`,
+  modalLeftMargin: `-10px`
 }
 
-const FlexRow = styled.div`
+const animation = {
+  slideDuration: `400`, // in ms
+  dimDuration: `400` // in ms
+}
+
+const Fonts = {
+  family: `'Nunito', sans-serif`,
+  small: `15px`,
+  medium: `17px`,
+  large: `18px`,
+  header: `25px`,
+  largeHeader: `30px`,
+  thin: `200`,
+  normal: `400`,
+  bold: `600`,
+  veryBold: `700`
+}
+
+const FlexRow = styled.div.attrs(props => {
+  return {
+    className: props.justify
+  }
+})`
 display: flex;
 flex-direction: row;
-margin-bottom: 2vh;
+margin-bottom: 1vh;
+
+&.center {
+  justify-content: center;
+}
+
+&.left {
+  justify-content: flex-start;
+  margin: 0 5vw;
+}
+
+`;
+
+const Text = styled.p.attrs(props => {
+  return {
```

Comment:
same thing here, can use ( ) instead of { return { } }

Reply:
^